CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+##[2.5.1] - 2018-05-08
```

Comment:
per our [changelog guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) we version and date when the submission was released not when it was submitted. We do this for several reasons:
- PRs are not a FIFO queue there is no guarantee on order in which they will be reviewed or merged in
- there is no SLA for reviews
- the maintainer may disagree with you on how the change should be versioned, in this case you have bumped it as a patch when this would be a minor version bump as it is adding new functionality rather than fixing existing. Please see our [versioning guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md) to understand how we evaluate change and update versions.
- in your case you have multiple PRs which the exact same version which would lead to the same version being in the changelog three different times which is not what we would want as it would lead to confusion even if we released all three changes in the same release.